jsf/injection/src/main/java/org/jboss/as/jsf/injection/AnnotationMap.java

```diff
@@ -70,7 +70,7 @@
             stringToAnnoMap.put(FacesBehavior.class.getName(), FacesBehavior.class);
             stringToAnnoMap.put(FacesBehaviorRenderer.class.getName(), FacesBehaviorRenderer.class);
 
-            // Put JSF 2.2+ annotations below this line if any new ones are to be scanned.
+            // Put Jakarta Server Faces 2.2+ annotations below this line if any new ones are to be scanned.
```

Comment:
@fjuma do you agree with changing `2.2` to `2.3`?

Reply:
We should leave this as is. See above.